### PR TITLE
add always ask if tagged test

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?

This PR adds a "test" to the framework (inverted commas cause, like the ask 4 earning "test", it is an earner rather than a learner). 

It's primary function is to allow us to turn the epic always on for **articles** or **interactives** that are tagged with any tag that is chosen in the targetting tool (https://targeting.gutools.co.uk/#).

It ignores the sensitive flag, so any articles tagged with and whitelisted tags will show the epic, regardless of whether they are sensitive or not. This is the desired behaviour, as it solves a probelm we have sometimes where an article is marked as sensitive for reasons that shouldn't stop us showing our asks on them.

**All** articles or interactives with the selected tags will **always** display the epic (until the tag is deselected in the targetting tool).

It also allows us to go into Hyper Mode™ when big events happen and we want to guarantee that certain stories display the epic. 

It will always display the control version of the epic. 

## Screenshots
This is what the control epic looks like currently:
control

### CODE REVIEW

**Review by joelochlann — CHANGES_REQUESTED** [review]
If you're expecting this to work for interactives, I think you'll need to change this bit:

https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js#L100

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ define([
                         testimonialMessage: 'I’m a 19 year old student disillusioned by an unequal society with a government that has stopped even pretending to work in my generation’s interests. So for the strength of our democracy, for the voice of the young, for a credible, independent check on the government, this contribution was pretty good value for money.',
                         testimonialName: 'Jack H.'
                     })
+
```
Comment (dominickendrick) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js]: remove ?

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-always-ask-if-tagged.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    'common/modules/commercial/contributions-utilities'
+], function (
+    contributionsUtilities
+) {
+    return contributionsUtilities.makeABTest({
+        id: 'AcquisitionsEpicAlwaysAskWithTag',
+        campaignId: 'epic_always_ask_with_tag',
+
+        start: '2017-05-25',
+        expiry: '2018-07-19',
```
Comment (dominickendrick) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-always-ask-if-tagged.js]: Should we set this to be a longer test if its going to be used generically ?

**Review by dominickendrick — APPROVED** [review]

**Comment by jranks123** [comment]
closed and replaced with this PR : https://github.com/guardian/frontend/pull/16913

**Pull request closed without merging**